codalab/worker/local_run/local_run_manager.py

```diff
@@ -126,7 +130,8 @@ def start(self):
         """
         self.load_state()
         self._image_manager.start()
-        self._dependency_manager.start()
+        if not self._shared_file_system:
+            self._dependency_manager.start()
```

Comment:
does it make sense to still have a dependency manager on shared filesystem? Say, if the filesystem has a size limit for example and some cleanup process is needed to not exceed it

Reply:
just realized shared fs means shared among not only workers but the server as well. 

do you think there'll be a use for a fs shared only among workers?

Reply:
I don't see a use case in the short term, since the main new deployment regime we're aiming for is the job-queue system where workers are expected to be relatively short lived, and caches aren't used that much.

That said, there might be longer term uses...